client/remoterepo.c

```diff
@@ -106,11 +206,409 @@ _handle_curl_cb(
     }
     return dwError;
 }
+
+
+uint32_t
+TDNF_Check_Digest(
+    const char *filename,
+    hash_op *hash,
+    uint8_t *result
+    )
+{
+    uint32_t dwError = 0;
+    int fd = -1;
+    hash_ctx_t ctx;
+    char buf[BUFSIZ] = {0};
+    int length = 0;
+
+    if(IsNullOrEmptyString(filename) ||
```

Comment:
Put these in one line. Looks a little odd.

Reply:
done.